src/styles/global.css

```
@@ -1207,6 +1207,89 @@
+    background: color-mix(in srgb, var(--surface) 92%, var(--muted) 8%);
+    color: var(--muted);
+  }
+
+  .place-card-trust-icon {
+    display: inline-flex;
+    width: 1rem;
+    height: 1rem;
+    flex: 0 0 1rem;
+    align-items: center;
+    justify-content: center;
+    color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)